lib/collection/url.js

```diff
@@ -13,6 +13,7 @@ var _ = require('../util').lodash,
     HTTPS_PORT = '443',
     HTTP_PORT = '80',
     PATH_SEPARATOR = '/',
+    PATH_VARIABLE_SEPARATOR = '.',
```

Comment:
non-blocker: Not sure about the variable name here. `.` isn't separating the path variables. It's just a character that shouldn't be part of the path key.